cfn/googlestackdriver_collector.template

```diff
@@ -0,0 +1,94 @@
+            "Type": "String",
+            "Default": "api.global-services.global.alertlogic.com",
+            "AllowedValues": [
+                "api.global-services.global.alertlogic.com",
+                "api.global-integration.product.dev.alertlogic.com"
+            ]
+        },
+        "AlDataResidency": {
+            "Description": "Alert Logic Data Residency",
+            "Type": "String",
+            "Default": "default",
+            "AllowedValues": ["default"]
+        },
+        "PackagesBucketPrefix": {
+            "Description": "S3 bucket name prefix where collector packages are located.",
+            "Type": "String",
+            "Default": "alertlogic-collectors"
+        },
+
+        "AlertlogicCustomerId": {
+            "Description": "Optional, Alert Logic customer ID which collected data should be reported for. If not set customer ID is derived from AIMs tokens",
+            "Type": "String",
+            "Default": ""
+        },
+        "CollectorId": {
+            "Description": "Optional, a collector UUID if known.",
+            "Type": "String",
+            "Default": "none"
+        },
+        "GCPKey": {
+            "Description": "Client secret for Auth0 authentication",
+            "Type": "String",
+            "NoEcho": true
+        },
+        "GoogleResourceIds": {
+            "Description": "JSON list of google resources to poll logs from. In the format <resourceType>/<resourceID>",
+            "Type": "String"
+        },
+        "CollectionStartTs": {
+            "Description": "Timestamp when log collection starts. For example, 2020-01-13T16:00:00Z",
+            "Type": "String",
+            "AllowedPattern" : "^\\d{4}(-\\d{2}){2}T(\\d{2}:){2}\\d{2}Z$"
+        }
+    },
+    "Environment":{
+       "Variables":{
+          "google_resource_ids":{
+              "Ref":"GoogleResourceIds"
+          }
+    },
+    "Resources":{
+      "Auth0CollectorStack" : {
+      "Type" : "AWS::CloudFormation::Stack",
+      "Properties" : {
+        "TemplateURL" : {"Fn::Join" : ["", [
+                    "https://s3.amazonaws.com/",
+                    {"Ref":"PackagesBucketPrefix"}, "-us-east-1",
+                    "/cfn/paws-collector.template"
+               ]]},
+        "Parameters" : {
+          "AlertlogicAccessKeyId" : { "Ref":"AlertlogicAccessKeyId" },
+          "AlertlogicSecretKey" : { "Ref":"AlertlogicSecretKey" },
+          "AlApiEndpoint" : { "Ref":"AlApiEndpoint" },
+          "AlDataResidency" : { "Ref":"AlDataResidency" },
+          "PackagesBucketPrefix" : { "Ref":"PackagesBucketPrefix" },
+          "PawsCollectorTypeName" : "googlestackdriver",
+          "AlertlogicCustomerId" : { "Ref":"AlertlogicCustomerId" },
+          "CollectorId" : { "Ref":"CollectorId" },
+          "PollingInterval" : 60,
+          "PawsEndpoint" : { "Ref":"Auth0Domain" },
+          "PawsAuthType" : "oauth2",
+          "PawsClientId" : { "Ref":"Auth0ClientId" },
```

Comment:
Auth0Domain, Auth0ClientId? Lease check for copy-paste

Reply:
(disappear)